test/e2e/density.go

```diff
@@ -631,6 +659,44 @@ var _ = framework.KubeDescribe("Density", func() {
 			}
 		})
 	}
+
+	// Calculate total number of pods from each node's max-pod
+	It("[Feature:ManualPerformance] should allow running maximum capacity pods on nodes", func() {
+		totalPods = 0
+		for _, n := range nodes.Items {
+			totalPods += int(n.Status.Capacity.Pods().Value())
+		}
+		totalPods -= framework.WaitForStableCluster(c, masters)
+
+		fileHndl, err := os.Create(fmt.Sprintf(framework.TestContext.OutputDir+"/%s/pod_states.csv", uuid))
+		framework.ExpectNoError(err)
+		defer fileHndl.Close()
+		rcCnt := 1
+		RCConfigs := make([]framework.RCConfig, rcCnt)
+		for i := 0; i < rcCnt; i++ {
+			RCName = "density" + strconv.Itoa(totalPods) + "-" + strconv.Itoa(i) + "-" + uuid
+			RCConfigs[i] = framework.RCConfig{Client: c,
+				Image:                "gcr.io/google_containers/pause-amd64:3.0",
+				Name:                 RCName,
+				Namespace:            ns,
+				Labels:               map[string]string{"type": "densityPod"},
+				PollInterval:         10 * time.Second,
+				PodStatusFile:        fileHndl,
+				Replicas:             (totalPods + rcCnt - 1) / rcCnt,
```

Comment:
It won't saturate the cluster unless `totalPods + rcCnt - 1` is for some reason divisible by `rcCnt` (it's OK for 1, but please make it work for other values as well).